Lib/defcon/objects/layer.py

```diff
@@ -206,6 +206,7 @@ def newGlyph(self, name):
         glyph.enableNotifications()
         self.postNotification("Layer.GlyphAdded", data=(dict(name=name)))
         self.dirty = True
+        return glyph
```

Comment:
It was already there: https://github.com/typesupply/defcon/blob/ufo3/Lib/defcon/objects/font.py#L187